CONTRIBUTING.md

```diff
@@ -1,37 +1,261 @@
-# Contributing to Coq
+# Guide to contributing to Coq #
 
-Thank you for your interest in contributing to Coq! There are many ways to
-contribute, and we appreciate all of them. Please make sure you read and
-abide by the [Code of Conduct](CODE_OF_CONDUCT.md).
+## Foreword ##
 
-## Bug Reports
+As any piece of code or documentation, this guide can bitrot if people
+forget to update it when contributing processes, development tools, or
+the Coq ecosystem changes.  If you notice anything inaccurate or
+outdated, please signal it in a new issue, or fix it in a new pull
+request.
```

Comment:
Oh, no, definitely not to a mailing list! In case you didn't notice, I'm all about reducing the space devoted to mailing lists in favor of more browsable, more searchable forms of discussions (like forums). In particular, I didn't even mention coqdev once in this guide because I want to kill it *short term* (in favor of the development category of the Discourse forum). Coq-Club is still bound to live much longer OTOH.